src/routes/api/[...endpoint]/+server.ts

```diff
@@ -11,3 +11,6 @@ export const GET: RequestHandler = dispatchToBackend;
 export const POST: RequestHandler = dispatchToBackend;
 export const PUT: RequestHandler = dispatchToBackend;
 export const DELETE: RequestHandler = dispatchToBackend;
+export const PATCH: RequestHandler = dispatchToBackend;
+export const OPTIONS: RequestHandler = dispatchToBackend;
+export const HEAD: RequestHandler = dispatchToBackend;
```

Comment:
Is adding `CONNECT` and `TRACE` as well worth it?

Reply:
Do not think so, they are not the type of requests we would do to the server via the frontend